packages/entity-generator/src/SourceFile.ts

```diff
@@ -498,6 +498,9 @@ export class SourceFile {
       assign('precision');
       assign('scale');
     }
+    if (prop.unsigned && this.platform.supportsUnsigned()) {
+      assign('unsigned');
+    }
```

Comment:
this is a general rule, all control statements like if/for/while should have blank lines around for better readability

```suggestion

    if (prop.unsigned && this.platform.supportsUnsigned()) {
      assign('unsigned');
    }

```

it's not super important, but since you are contributing more and more (and the rest of the codebase is written this way), please try to adhere to this